web/src/server_events_dispatch.js

```diff
@@ -904,19 +904,6 @@ export function dispatch_normal_event(event) {
                 );
                 activity_ui.build_user_sidebar();
             }
-            if (event.property === "dense_mode") {
-                $("body").toggleClass("less-dense-mode");
-                $("body").toggleClass("more-dense-mode");
-                information_density.set_base_typography_css_variables();
-                information_density.calculate_timestamp_widths();
-            }
-            if (
-                event.property === "web_font_size_px" ||
-                event.property === "web_line_height_percent"
-            ) {
-                information_density.set_base_typography_css_variables();
-                information_density.calculate_timestamp_widths();
-            }
```

Comment:
Let's replace this with an explicit noop block, rather than deleting it.

Reply:
There was actually a mistake here where the value in `user_settings` was being updated without this functions be called. So, fixed that and added a comment as well.